experimental/token/cursor.go

```diff
@@ -76,8 +76,8 @@ func (c *Cursor) IsSynthetic() bool {
 // Clone returns a copy of this cursor, which allows performing operations on
 // it without mutating the original cursor.
 func (c *Cursor) Clone() *Cursor {
-	copy := *c
-	return &copy
+	c2 := *c
```

Comment:
How about call it `clone`? I'm generally not a fan of numeric suffixes to distinguish two variable names. They aren't meaningful so aren't very helpful.